tsdb/wal/watcher.go

```diff
@@ -483,6 +488,10 @@ func (w *Watcher) readSegment(r *LiveReader, segmentNum int, tail bool) error {
 			}
 			for _, s := range samples {
 				if s.T > w.StartTime {
+					if !sentSamples && segmentNum == w.lastSegment {
+						sentSamples = true
+						level.Info(w.logger).Log("msg", "done replaying WAL")
```

Comment:
Do you think it is worthwhile to log the elapsed time here rather than having to look for two log entries?

Reply:
yep!